.github/workflows/ci-tr.yml

```
@@ -0,0 +1,103 @@
+      #- '.github/**'
+      - '!.github/workflows/ci.yml'
+    branches:
+      - 'trunk'
+      - 'release/*'
+      - 'tR-e2e-tests'
+  workflow_call:
+    inputs:
+      trigger:
+        description: 'Type of run to trigger. E.g. nightly-checks, release-checks, etc.'
+        required: true
+        default: 'default'
```

actionlint (1.7.7)

input "trigger" of workflow_call event has the default value "default", but it is also required. if an input is marked as required, its default value will never be used

(events)